src/app.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+const operation = process.argv[2];
+const numbers = process.argv.slice(3);
+let res;
+
+if (operation === 'add' && numbers.length > 0) {
+  res = numbers.reduce((acc, cur) => {
+    return Number(acc) + Number(cur);
+  }, 0);
+} else if (operation === 'multiply' && numbers.length > 0) {
+  res = numbers.reduce((acc, cur) => {
+    return Number(acc) * Number(cur);
+  }, 1);
+} else if (!operation) {
```

Comment:
we may move this in the beginning of function, bcz if not supported why we need check in others if? that we may do with next condition in if too.